### ISSUE TITLE
./vendor/bin/yii: No such file or directory

### ISSUE THREAD

**Issue by machour** [yiisoft/yii-base-web#60]
### What steps will reproduce the problem?

Installed this template following the README instructions:

```
composer create-project --prefer-dist --stability=dev yiisoft/app myapp
cd myapp
./vendor/bin/yii serve
```

### What's expected?

The app to be served

### What do you get instead?

For last step:

```
./vendor/bin/yii: No such file or directory
```

### Additional info

My `vendor/bin/` directory only contains `phpunit`

The only `yii` binary found within the tree is:
```
~/myapp/vendor $ find . -name yii
./yiisoft/yii-core/contrib/completion/bash/yii
``` 

| Q                | A
| ---------------- | ---
| Yii version      | 3 ?
| PHP version      | 7.1.19
| Operating system | macOS Darwin Kernel Version 18.2.0

**Comment by machour** [comment]
I thought this was the starting point to create a new yii3 web app, but it seems that I have to install `yii-project-template` and then require the web part?

I know that this is still work in progress, but this is **really confusing** (and I've been on yii1/yii2 for years).
For example, this repository README is just wrong (from my understanding).

What's confusing me the most is that there don't seem to be a single page, up to date, with **all** that I need to know before jumping into the yii3 wagon. 

I've been already pointed to the forums, but forums posts tend to be confusing, cluttered with q&a, and (maybe?) already outdated when I get to read them.

While I know that the priority is development here, rather than documentation, and that things are still changing drastically, I think that if the core-team could maintain a single wiki page, stating all that we need to know for yii3, a lot of confusion would vanish.

Current resources that could go on this page:

- https://forum.yiiframework.com/t/how-to-get-started-with-yii-3-0-application/123955
- https://github.com/yiisoft/docs/blob/master/000-packages.md

**Comment by samdark** [comment]
You can start it at https://www.yiiframework.com/wiki... or at a forum.

**Comment by machour** [comment]
Started something at https://www.yiiframework.com/wiki/2547/draft-understanding-yii-3
Could you have a look and maybe correct my wrong assumptions ?

Next step I'd document would be adding a step by step installation & customization of a web project using `yii-base-web`, but I'm still trying to get a hang of things (still [struggling](https://github.com/yiisoft/yii-dataview/issues/8) with GridView & DI)

**Comment by schmunk42** [comment]
> For example, this repository README is just wrong (from my understanding).

Yes, it totally needs an update...

**Comment by machour** [comment]
Fixed in #62
